Add Documentation for the JSM Integration (#1636)
OctopusDeploy/docs: docs/approvals/index.md

> hideInThisSection: true

Reviewer: If you remove the section header, and not the entire contents, the page becomes more dynamic
```suggestion
hideInThisSectionHeader: true
```

---

> Octopus currently supports blocking deployments for a project and environment based on the 
> approval status of a Change Request in a supported ITSM system. Currently, both ServiceNow and 
> Jira Service Desk are supported.

Reviewer: ```suggestion
Octopus Deploy includes dedicated integrations for popular ITSM systems to help support blocking deployments for a project and environment based on the approval status of a Change Request.
```

> For more information, read our documentation on [Service Now Integration](service-now/index.md) 
> and [Jira Service Management Integration](jira-service-management/index.md).

Reviewer: Removing this allows for the child page selector to show the child pages for you as they expand over time :) 
```suggestion
```
